src/github/issue.ts

```diff
@@ -16,10 +13,7 @@ export const issueWebhookHandler = async (context: CustomContext<WebhookPayloadI
 		}
 	} = context.payload;
 
-	const gitHubBaseUrl = await getGitHubBaseUrl(jiraHost);
-	const githubClient = await gheServerAuthAndConnectFlowFlag(jiraHost)
-		? new GitHubInstallationClient(getCloudInstallationId(githubInstallationId, gitHubBaseUrl), context.log, gitHubBaseUrl)
-		: new GitHubInstallationClient(getCloudInstallationId(githubInstallationId), context.log);
+	const githubClient = await createInstallationClient(githubInstallationId, jiraHost, context.log);
```

Comment:
Consolidated the creation of an installation client in `createInstallationClient` (I didn't replace all occurrences, this is just an example)